integration/shareddata/scalars.go

```diff
@@ -134,20 +138,22 @@ var Doubles = &Values[string]{
 		"double-big-plus":  doubleBig + 1,
 		"double-big-minus": doubleBig - 1,
 
-		// double max precision ~1<<53
-		"double-prec-max":       doubleMaxPrec,
-		"double-prec-max-plus":  doubleMaxPrec + 1,
-		"double-prec-max-minus": doubleMaxPrec - 1,
+		// double max precision ~1<<53 - 1
+		"double-prec-max":          doubleMaxPrec,
+		"double-prec-max-plus":     doubleMaxPrec + 1,
+		"double-prec-max-plus-two": doubleMaxPrec + 2,
+		"double-prec-max-minus":    doubleMaxPrec - 1,
 
 		// negative double big values ~ -(1<<61)
 		"double-neg-big":       -doubleBig,
-		"double-neg-big-plus":  -(doubleBig + 1),
-		"double-neg-big-minus": -(doubleBig - 1),
+		"double-neg-big-plus":  -doubleBig + 1,
+		"double-neg-big-minus": -doubleBig - 1,
 
 		// double min precision ~ -(1<<53 - 1)
-		"double-prec-min":       -(doubleMaxPrec - 1),
-		"double-prec-min-plus":  -(doubleMaxPrec - 1) + 1,
```

Comment:
Where is `double-prec-min-plus`? It should `-doubleMaxPrec+2` but seems to be missing?

Reply:
There's a `double-prec-min-plus` at line 154.

Reply:
I know the same variable name `double-prec-min-plus` exists. What I want to know is why the value of the variable has changed. It was  `-(doubleMaxPrec - 1) + 1` which is equivalent of `-doubleMaxPrec+2` right? Now it is `-doubleMaxPrec + 1` which is the different value.

Reply:
Okay so this is for the same reason we don't need both +2 because it's tested by -1, and -2.

Reply:
The previous value was wrong.
`double-prec-min` is not `-(1<<53 -1) but just `-(1<<53)`